src/gsCore/gsGeometry.hpp

```diff
@@ -129,11 +129,19 @@ void gsGeometry<T>::evaluateMesh(gsMesh<T>& mesh) const
 
     // For all vertices of the mesh, push forward the value by the
     // geometry mapping
-    for (size_t i = 0; i!= mesh.numVertices(); ++i)
-    {
-        eval_into( mesh.vertex(i).topRows(pDim), tmp );
-        mesh.vertex(i).bottomRows( gDim ) = tmp;
-    }
+    if (1==gDim && 3<pDim) // Plot a graph
+        for (size_t i = 0; i!= mesh.numVertices(); ++i)
+        {
+            eval_into( mesh.vertex(i).topRows(pDim), tmp );
+            mesh.vertex(i).middleRows(pDim, gDim) = tmp;
+        }
+    else // Plot mesh on a mapping
+        for (size_t i = 0; i!= mesh.numVertices(); ++i)
+        {
+            eval_into( mesh.vertex(i).topRows(pDim), tmp );
+            mesh.vertex(i).topRows(gDim) = tmp;
+        }
+
```

Comment:
What was wrong with using `bottomRows`? I tested it for
f: [0,1]^2 -> R^1
and
f: [0,1]^2 -> R^3
and it worked in both cases.

The `middleRows` solution gives a wrong result in the test case from issue #348, at least on my computer.

Reply:
I think that bottomRows would not be correct for case
 f: [0,1]^1 -> R^1
Essentially, middleRows(pDim, gDim) should be the same result as bottomRows(gDim) for the case #348, I will check again..

Reply:
I was also surprised. My guess was that `middleRows(pDim+1, gDim)` would be the same as the `bottomRows` but even that did not behave correctly. It is of course possible that I have made a mistake somewhere on the way but I have double-checked and found nothing wrong.

Reply:
One more thought occurred to me: what is actually the expected outcome for
f: [0,1]^1 -> R^1?

What is the element mesh supposed to depict? Does it make sense to do something like
`gsWriteParaview(fn, string, 100, true)` at all?